src/components/ReportActionItem/ReportPreview.js

```diff
@@ -125,7 +125,7 @@ const ReportPreview = (props) => {
                     />
                 </Pressable>
             ))}
-            {isCurrentUserManager && props.iouReport.stateNum === CONST.REPORT.STATE_NUM.PROCESSING && (
+            {isCurrentUserManager && props.iouReport.state === CONST.REPORT.STATE.PROCESSING && (
```

Comment:
`CONST.REPORT.STATE.PROCESSING` doesn't seem to be defined

Reply:
Oh you are right! Stupid PHPStorm was jumping to STATE_NUM.PROCESSING when I clicked, so I thought it was.